### ISSUE TITLE
no notification for incoming msgs [Custom HS]

### ISSUE THREAD

**Issue by ylecollen** [vector-im/riot-android#1381]
reported by @przemas75:www.anuszek.net
https://github.com/matrix-org/riot-android-rageshakes/issues/446
https://github.com/matrix-org/riot-android-rageshakes/issues/445

**Comment by ylecollen** [comment]
It works properly on my side.

The GCM registration worked properly but the GCM method onMessageReceived is not called.

**Comment by ylecollen** [comment]
I asked to the user to check if the notifications are triggered when the application is in foreground to check if this issue is triggered because there is no notification which matches or the server failed to send pushes.

**Comment by ylecollen** [comment]
@przemas75:www.anuszek.net : any news on your server side.

**Comment by przemas75** [comment]
I tried to play with battery/power settings on both phones. I read somewhere while riot goes to sleep android "kills" it somehow ( i dont know). 
So, on android 6 i got 'no battery optimalization' for Riot. On the other phone (android 7) i dont recall setting name, but the same idea.
I think, the issue can be closed.
I saw for syncthing android application question during installation about battery setting.
Maybe the same would be useful for Riot?

Thanks for all!
